components/avatar/avatar.css

```diff
@@ -0,0 +1,42 @@
+* {
+  font-size: 110%;
+}
+
+.avatar {
+  border-radius: 100%;
+  margin-bottom: 2rem;
+}
+
+.sqr {
+  border-radius: 5px;
+}
+
+.xs {
+  width: 24px;
+  height: 24px;
+}
+
+.sm {
+  width: 32px;
+  height: 32px;
+}
+
+.md {
+  width: 48px;
+  height: 48px;
+}
+
+.lg {
+  width: 64px;
+  height: 64px;
+}
+
+.xl {
+  width: 96px;
+  height: 96px;
+}
+
+.xxl {
```

Comment:
The naming of these classes could be better. Because if you have some other component having a similar variant of small, large, and medium the naming could cause overwriting the CSS.